CHANGELOG.md

```diff
@@ -7,31 +7,26 @@ rocBLAS documentation is available at
 
 ### Added
 
-### Changed
-
-### Removed
-
-* `rocblas_hgemm_kernel_name`, `rocblas_sgemm_kernel_name`, and `rocblas_dgemm_kernel_name` API functions
-
-### Optimized
-
-### Resolved issues
-
-## (Unreleased) rocBLAS 4.5.0
-
-### Added
-
 * `ROCBLAS_LAYER = 8` internal API logging for `gemm` debugging
 * Support for AOCL 5.0 gcc build as a client reference library 
 * Allow `PkgConfig` for client reference library fallback detection 
+* gfx950 support
```

Comment:
I would put gfx950 support first for emphasis

Reply:
Moved to be first